### DISCUSSION TITLE
创建Nodejs运行环境时报错

### DISCUSSION THREAD

**Discussion by lijing-2008** [1Panel-dev/1Panel#6009] [Q&A]
### 联系方式

huyispider@gmail.com

### 1Panel 版本

v1.10.13-lts

### 问题描述

在创建nodejs运行时环境时  报错: 服务器内部错误：gzip: invalid header
![error](https://github.com/user-attachments/assets/92441fda-b770-4a95-b1aa-6cf3748868c2)

### 重现步骤

就是简单的创建一个运行时环境而已

**Comment by wanghe-fit2cloud** [comment]
进去 `主机` - `终端` 页面，执行 `tar --version` 命令检查一下 tar 命令版本

**Reply by lijing-2008** [reply]
执行过了，显示如下：
tar (GNU tar) 1.34
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by John Gilmore and Jay Fenlason.

**Reply by lijing-2008** [reply]
并且我选择我的文件目录，竟然一直提示目录不存在，好奇怪
![Longshot-20240802154724](https://github.com/user-attachments/assets/f8566be7-9c64-49c8-9fd7-3f466ed37171)
![Longshot-20240802154758](https://github.com/user-attachments/assets/75f7c394-b7dc-4688-85c8-93915c7b7e0a)

**Reply by zhengkunwang223** [reply]
![image](https://github.com/user-attachments/assets/0d72a2d6-b792-48bc-995b-6b2abaf2ca24)
需要勾选目录 不能直接点确定

**Reply by wanghe-fit2cloud** [reply]
我们将在后续版本中优化文件夹选中逻辑，具体进展请关注此 [issue](https://github.com/1Panel-dev/1Panel/issues/6012)

**Reply by lijing-2008** [reply]
我创建nodejs环境后日志里没有反应，然后创建结果是失败，看了下是docker相关问题，如下图，我之前一直正常用的，现在一直创建不起来，不知道什么问题，docker源我也重新配置过了，请教现在有什么解决方案。
![Longshot-20240807064417](https://github.com/user-attachments/assets/ad9aa277-1683-4191-9450-3f38d81a4d98)

**Reply by wanghe-fit2cloud** [reply]
看错误信息应该还是拉取基础镜像超时导致的。
你有配置镜像加速吗？
![image](https://github.com/user-attachments/assets/d9b3cc57-8c6b-43bf-90d9-c175cd0596b7)
